### ISSUE TITLE
All gene entities want to be mrna or mrna_contig

### ISSUE THREAD

**Issue by bradfordcondon** [statonlab/hardwoods_site#90]
Convert from gene to mrna or mrna_contig.

Hydrangea wants to be mrna_contig... look at the analysis type, trnascriptomes want to be mrna_contig, if its genome its mrna

**Comment by bradfordcondon** [comment]
Alchemist will currently allow me to batch convert *ALL* of them.  or do them individually by hand.

If *all genes* should be one type, I can go ahead.

If some need to be one type, and some the other, then I need to build more functionality into alchemist first (to differentiate based on existing property)

**Comment by bradfordcondon** [comment]
There are four organisms with features set to gene (type_id 215)

* 24 - C. mollissima
* 37 C. mollissima (QTL regions)
* 44 - C. florida (dogwood)
* 45 - H. macrophylla

**Comment by bradfordcondon** [comment]
For transcriptomes:
```
UPDATE chado.feature SET type_id='101394' WHERE type_id = '215' AND organism_id = '';
```

For genomes:
```
UPDATE chado.feature SET type_id='145' WHERE type_id = '215' AND organism_id = '';
```

**Comment by bradfordcondon** [comment]
Alchemist isn't ready for this, i forgot (uses type_id).

Instead we're going to just update chado.feature.type_id, delete the gene bundle, and re-migrate features.  this should work as planned and result in no duplicates.

**Comment by bradfordcondon** [comment]
UPDATE chado.feature SET type_id=145 WHERE type_id = 215 AND organism_id = 24;

UPDATE chado.feature SET type_id=101394 WHERE type_id = 215 AND organism_id = 37;
UPDATE chado.feature SET type_id=101394 WHERE type_id = 215 AND organism_id = 44;
UPDATE chado.feature SET type_id=101394 WHERE type_id = 215 AND organism_id = 45;

**Comment by bradfordcondon** [comment]
NOTE:  `UPDATE chado.feature SET type_id=101394 WHERE type_id = 215 AND organism_id = 37;` only had 747 features????????

**Comment by almasaeed2010** [comment]
Which organism is this?

**Comment by bradfordcondon** [comment]
- [ ] c mollisima qtl 

- [ ] c mollisima 

- [ ] c florida

- [ ] h macrophylla

**Comment by bradfordcondon** [comment]
all published!

Fix the URL'S and we're golden!  #118
